src/plots/linePlot.ts

```diff
@@ -27,8 +27,42 @@ type EdgeIntersections = {
   bottom: Point[]
 };
 
+const CURVE_NAME_MAPPING: { [name: string]: d3.CurveFactory | d3.CurveFactoryLineOnly } = {
+  "linear": d3.curveLinear,
+  "linear-closed": d3.curveLinearClosed,
+  "step": d3.curveStep,
+  "step-before": d3.curveStepBefore,
+  "step-after": d3.curveStepAfter,
+  "basis": d3.curveBasis,
+  "basis-open": d3.curveBasisOpen,
+  "basis-closed": d3.curveBasisClosed,
+  "bundle": d3.curveBundle,
+  "cardinal": d3.curveCardinal,
+  "cardinal-open": d3.curveCardinalOpen,
+  "cardinal-closed": d3.curveCardinalClosed,
+  "monotone": d3.curveMonotoneX,
```

Comment:
This is so much more sane than that easing function hackery above. Can we do something like this for easing functions?